imap_processing/ultra/l1b/cullingmask.py

```diff
@@ -25,14 +26,31 @@ def calculate_cullingmask(
     cullingmask_dataset : xarray.Dataset
         Dataset containing the data.
```

Comment:
Again, I think it would be good to add more explanation here to the Dataset description.